### ISSUE TITLE
Certificate Management

### ISSUE THREAD

**Issue by jseiser** [stackrox/stackrox#11718]
Is there a reason that the init bundle is required?  Its very cumbersome for our workflow to have to run roxctl, generate certs then feed it to a second helm chart.

1. I would expect to be able to handle my certificates via cert-manager. Create a CA, create a issuer, and then bundle it all up and deploy it into the stackrox namespace.
2. If I can not handle certs via cert-manager, I would expect to be able to load the certificates from rox init as a k8s secret and reference them in the helm chart.

**Comment by gavin-stackrox** [comment]
Hi @jseiser the init-bundle is required for secure `SecuredCluster` to `Central` communication. You only have to generate it once per `Central` instance if that helps: https://docs.openshift.com/acs/4.4/installing/installing_other/init-bundle-other.html

**Comment by jseiser** [comment]
Im more asking why we are required to generate things out of band, instead of using what is generally a pretty well documented process, e.g., cert manager, or mounting certificates as secrets.

In a git ops workflow, having to handle this init bundle stuff, ruins that work flow.

**Comment by gavin-stackrox** [comment]
Yes, this is a known issue for improvement. [this is probably not a public link, but leaving here for reference: https://issues.redhat.com/browse/ROX-18675]

**Comment by acravn** [comment]
I have a fix for this in a forked version of the helm charts. It supports pre-creating TLS secrets ahead of time and passing in those secret names to the values file. I do still pull the init bundle to generate the secret, but that doesn't stop someone from creating valid cert/keys and pre-creating the secret. I'm working on getting a PR setup to push it with a couple other values file related changes my org needed.
